nodeBackend/README.md

```
@@ -30,4 +30,6 @@
 ├─ .env                            # local dev env (not committed secrets)
 └─ README_BACKEND.md
 
+video-worker: decided to be in flask itself never to be migrated to JS!
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)